packages/state-history/src/lib/entities-state-history.spec.ts

```diff
@@ -0,0 +1,410 @@
+import { Store, createStore, withProps } from '@ngneat/elf';
```

Comment:
'Store' is defined but never used

Reply:
It's used but as type

Reply:
Oh, It's in a `spec`. Yes, it's an extra and I'' delete it